AirportDepartures.playground/Contents.swift

```diff
@@ -16,8 +16,56 @@ import UIKit
 //: e. Use a `String?` for the Terminal, since it may not be set yet (i.e.: waiting to arrive on time)
 //:
 //: f. Use a class to represent a `DepartureBoard` with a list of departure flights, and the current airport
-
-
+enum FlightStatus: String {
+    case EnRoute_Delayed
+    case EnRoute_On_Time
+    case Landed_Delayed
+    case Landed_On_Time
+    case Landed
+    case Scheduled
+    case Cancelled
+    
+}
+
+struct Airport {
+    let destination: String
+    let arrival: String
+}
+
+struct Flight {
+    var flightNumber: String
+    var destination: String
+    var airline: String
+    var terminal: Int?
+    var departure: Date?
+    var status: FlightStatus
+}
+
+class DepartureBoard {
+    var departingFlights: [Flight]
+    
```

Comment:
The departure board should have an additional variable--"airport". I should have pointed that out earlier.